todolist/android/gradle.properties

```diff
@@ -16,3 +16,7 @@
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
+TODOLIST_RELEASE_STORE_FILE=todolist-key.keystore
```

Comment:
okey.. first, you should't commit this to github. This are used to sign the apk, so is kind of an important key. We try to never add keys or similar to git. Also.. this credentials are in the wrong place, we should use them in a system file not in the project. We will check this in a call.. just delete this line and we are good